scripts/ci/build_emscripten.sh

```diff
@@ -40,6 +40,8 @@ else
 	BUILD_DIR="$1"
 fi
 
+apt-get update && apt-get install lz4 wabt binaryen
```

Comment:
Once we're sure we want to go for this, we should move this to the Dockerfile instead.

Reply:
I suggest to do this in a second step after merging this PR.